perllib/FixMyStreet/Script/Merton/SendWaste.pm

```diff
@@ -17,6 +18,7 @@ sub send_reports {
     my $problems = $self->_problems;
 
     while (my $row = $problems->next) {
+        $self->set_echo_id($row) or next; # skip if no echo_id
```

Comment:
If Echo returns an error I figured it'd be better to not send to Crimson, as it might just have been a transient error and the next attempt would work OK.